src/server/pachyderm_test.go

```diff
@@ -10625,6 +10625,52 @@ func TestPodPatchUnmarshalling(t *testing.T) {
 	}
 }
 
+func TestSecrets(t *testing.T) {
+	if testing.Short() {
+		t.Skip("Skipping integration tests in short mode")
+	}
+
+	c := getPachClient(t)
+	require.NoError(t, c.DeleteAll())
+	c.DeleteSecret("test-secret", "default")
```

Comment:
I'm assuming this is cleanup so the test can be run multiple times in a row since the secrets don't get deleted by `DeleteAll`, I think it probably makes sense to have `DeleteAll` delete the secrets as well. Obviously it shouldn't delete other secrets in the namespace, but the ones that were created by Pachyderm make sense I think. The easiest way to implement that would probably be to have a label that you apply to the secrets that indicates that they were created by pachyderm, and then I think you can just target the deletion using a selector on those labels. We do this for `pachctl undeploy` if you want to see an example.